src/features/expenses-list/expenses-list.routes-mock.ts

```diff
@@ -0,0 +1,71 @@
+import { Router } from "express";
+import { promises as fs } from "fs";
+import path from "path";
+export const router = Router();
+
+router.get("/", async (req, res) => {
+  let jsonPath = path.join(
+    __dirname,
+    "..",
+    "..",
+    "mock",
+    "expensesListsMock.json"
+  );
+
+  try {
+    const data = await fs.readFile(jsonPath, "utf8");
+    let expensesListData = JSON.parse(data);
+
+    expensesListData.expenseLists = expensesListData.expenseLists.map(
+      (list: { expenses: any[] }) => {
+        // console.log(list.expenses);
+        const expenseTotal = list.expenses.reduce(
+          (sum: any, { price }: any) => sum + price,
```

Comment:
use number

Reply:
the file not necessery anymore, maybe will add again in the test branch